packages/devtools/web/main.dart

```diff
@@ -4,68 +4,98 @@
 
 import 'dart:async';
 import 'dart:html';
+
 import 'package:devtools/src/framework/framework_core.dart';
 import 'package:devtools/src/main.dart';
 import 'package:devtools/src/ui/analytics.dart' as ga;
 import 'package:devtools/src/ui/analytics_platform.dart' as ga_platform;
 import 'package:platform_detect/platform_detect.dart';
 
 void main() {
-  // Need to catch all Dart exceptions - done via an isolate.
-  runZoned(() {
-    // Initialize the core framework.
-    FrameworkCore.init();
+  // Run in a zone in order to catch all Dart exceptions.
+  runZoned(
+    () {
```

Comment:
Did dartfmt do this?

Reply:
yes because the tests are passing and the fail on bad formatting. :)